components/automate-cli/cmd/chef-automate/status.go

```diff
@@ -47,6 +78,24 @@ func newStatusCmd() *cobra.Command {
 
 	return statusCmd
 }
+func newStatusSummaryCmd() *cobra.Command {
+	var statusSummaryCmdFlags = StatusSummaryCmdFlags{}
+	var statusSummaryCmd = &cobra.Command{
+		Use:   "summary",
+		Short: "Retrieve Chef Automate status-summary",
+		Long:  "Retrieve Chef Automate status node summary for HA deployment",
+		RunE:  runStatusSummaryCmdFunc(&statusSummaryCmdFlags),
+	}
+	statusSummaryCmd.PersistentFlags().StringVarP(&statusSummaryCmdFlags.automateIp, "automate-ips", "A", "", "Get automate Status by ip addresses")
+	statusSummaryCmd.PersistentFlags().BoolVarP(&statusSummaryCmdFlags.isAutomate, "automate", "a", false, "Get only automate Status")
```

Comment:
Can you use flags that are already supported on automate-ha. Creating new flags will create confusion for customers.
You can use `chef-automate cert-rotate` command for reference as it supports flags for nodetypes as well as specific node ip.